test/contracts/staking/staking-factory.test.ts

```diff
@@ -42,26 +62,187 @@ export type StakingPoolLibData = {
     maxTotalPoolStake: number
 }
 
-describe('Staking Pool Factory', () => {
+describe.only('Staking Pool Factory', () => {
```

Comment:
issue: `describe` not `describe.only`

_An extra dot only_

Reply:
yeah this is a hack i use for dev to run tests faster. this PR should have been marked WIP, apologies if wasted cycles!